index.html

```diff
@@ -109,46 +145,52 @@ <h3 class="tag">2015</h3> </div>
      </ul>
       </div>
       <div class="project-check"> <button type="button">See Project</button> </div> 
+    </div>
    </div>
 
-   <div class="card-4">
-   <img src="Images/Snapshoot-Portfolio-image-4.png" alt="" class="card__image">
-
+   <div class="work-card card-4">
+   <img src="Images/work-card-art-prog.svg" alt="" class="card_image">
+   <div class="card-content">
    <h2 class="big-txt">Multi-Post Stories</h2>
+   <h2 class="replace-tonic">Multi-Post Stories</h2>
    <div class="tag-group">
     <h3 class="tag-Canopy">CANOPY</h3>
+    <h3 class="replace-canopy">FACEBOOK</h3>
     <div class="dot-separator"></div>
     <h3 class="tag">Back End Dev</h3>
+     <h3 class="replace-tag">Full Stack Dev</h3>
     <div class="dot-separator"></div>
-    <h3 class="tag">2015</h3> </div>
-    <div class="card-txt"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+    <h3 class="tag-year">2015</h3>
+    <h3 class="other-year">2015</h3> </div>
+    <p class="card-txt">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <p class="replace-p">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
 
     <div class="prog">
       <ul>
         <li>html</li> 
+        <li class="add-ruby">Ruby on rails</li>
         <li>css</li> 
         <li>javaScript</li>
      </ul>
       </div>
 
       <div class="project-check"> <button type="button">See Project</button> </div> 
+    </div>
   </div>
 
    </section>
 
-  <section class="headline-2">
+  <section class="all-about-myself">
```

Comment:
Kindly note that in the "About Myself" Section, the left block takes 40% and the right one the remaining 60% of the page's width 🤗 as shown below
Also, note that the top of About myself should align with the top of Languages as shown by the red line below 😃

Your design:
![image](https://user-images.githubusercontent.com/73714615/186941562-50f8843d-85ff-488f-ae9b-6447414495c6.png)

Figma design:
![image](https://user-images.githubusercontent.com/73714615/186941374-f7783ae9-28db-40b9-81f2-bf7f91fd77c5.png)